src/main/java/frc/robot/subsystems/drive/DriveSubsystem.java

```diff
@@ -32,7 +32,7 @@ public DriveSubsystem() {
         driveModules[1] = new SwerveModules(DriveConstants.frontRightDriveID, DriveConstants.frontRightRotationMotorID,
         DriveConstants.frontRightRotationEncoderID, DriveConstants.frontRightAngleOffset, true, false);
         driveModules[2] = new SwerveModules(DriveConstants.backRightDriveID, DriveConstants.backRightRotationMotorID,
-        DriveConstants.backRightRotationEncoderID, DriveConstants.backRightAngleOffset, false, false);
+        DriveConstants.backRightRotationEncoderID, DriveConstants.backRightAngleOffset, false, true);
```

Comment:
@redPlover This change inverts the turning mode - I meant that we should invert the drive mode (`true, false`). I should have been more clear: Since we are flipping the motor 180 degrees, we will be driving backwards from the old setting

Reply:
Done